### ISSUE TITLE
React 17 support

### ISSUE THREAD

**Issue by devopsix** [fraserxu/react-dropdown#213]
When using react-dropdown@1.9.1 in a project that depends on react@17.0.1 and react-dom@17.0.1 `npm install` issues peer dependency warnings:

```
npm install
npm WARN react-dropdown@1.9.1 requires a peer of react@^0.14.7 || ^15.0.0-0 || ^16.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-dropdown@1.9.1 requires a peer of react-dom@^0.14.7 || ^15.0.0-0 || ^16.0.0 but none is installed. You must install peer dependencies yourself.
```

Would you mind adding React 17 to the peer dependecies?

**Comment by devopsix** [comment]
PR #214

**Comment by fraserxu** [comment]
Thanks @devopsix Published to 1.9.2
